apps/performance-tests/src/hierarchies/StatelessHierarchyProvider.ts

```diff
@@ -71,30 +95,35 @@ export class StatelessHierarchyProvider {
     });
   }
 
-  private createECSchemaProvider() {
-    const iModel = this._props.iModel;
+  private static createNewECSchemaProvider(iModel: IModelDb) {
```

Comment:
why was this renamed? The "new" word seems redundant - "create" already says it's going to be new

Reply:
Renamed back to `createECSchemaProvider`